common/build/eslint-config-fluid/strict.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
 		es2024: false,
 		node: true,
 	},
-	extends: ["./recommended.js"],
+	extends: ["./recommended.js", "biome"],
```

Comment:
Do not merge without reverting this.

Reply:
I made this change so it was easier to see the diff between the strict and strict-biome configs via the printed config. Once reviewed I'll revert.